livereduce.spec

```diff
@@ -19,7 +19,8 @@ Url: https://github.com/mantidproject/livereduce
 BuildRequires: python%{python3_pkgversion} python%{python3_pkgversion}-setuptools
 
 Requires: python%{python3_pkgversion}
-Requires: python3-inotify
+Requires: jq
+Requires: nsd-app-wrap
```

Comment:
I thought this `Requires` only applied to packages retrievable with `yum`. Is this only a declaration of requirements without any validation